typing/types.mli

```diff
@@ -498,3 +503,11 @@ type label_description =
     lbl_loc: Location.t;
     lbl_attributes: Parsetree.attributes;
   }
+
+(** Extracts the list of "value" identifiers bound by a signature.
+    "Value" identifiers are identifiers for signature components that
+    correspond to a run-time value: values, extensions, modules, classes.
+    Note: manifest primitives do not correspond to a run-time value! *)
```

Comment:
I know this is copied over from another file, but I'd like to know what "manifest primitives" means. Thanks.